app/Importer/Importer.php

```diff
@@ -551,4 +552,35 @@ public function fetchManager($user_manager_first_name, $user_manager_last_name)
 
         return null;
     }
+
+    /**
+     * Parse a date or return null
+     *
+     * @author A. Gianotto
+     * @since 7.0.0
+     * @param $field
+     * @param $format
+     * @return string|null
+
+     */
+    public function parseOrNullDate($field, $format = 'date') {
+
+        $format = 'Y-m-d';
```

Comment:
This is immediately overwriting the provided `$date` so the following `if` check will never be truthy.

Reply:
I think $format there should be `$date_format`